packages/components/src/icon/index.tsx

```diff
@@ -68,10 +68,10 @@ function Icon< P >( {
 
 	if ( 'function' === typeof icon ) {
 		if ( icon.prototype instanceof Component ) {
-			return createElement( icon, ( {
+			return createElement( icon, {
 				size,
 				...additionalProps,
-			} as unknown ) as P );
+			} as unknown as P );
```

Comment:
Parenthesis removed

Reply:
Yes, that's what the updated Prettier does to all TypeScript `X as Y as Z` expressions. There are no longer any parens.